dist/tools/tapsetup/tapsetup

```diff
@@ -33,6 +33,16 @@ unsupported_plattform() {
     echo "(currently supported \`uname -s\` 'Darvin', 'FreeBSD', and 'Linux')" >&2
 }
 
+update_uplink() {
+    if command -v dhclient > /dev/null; then
+        dhclient $1
+    else
+        echo "DHCP client `dhclient` not found." >&2
+        echo "Please reconfigure your DHCP client for interface ${BRNAME}" >&2
+        echo "to keep up-link's IPv4 connection." >&2
```

Comment:
IPv4?

Reply:
Yes, IPv6 connectivity keeps working through ~~magic~~ router advertisements?.
It's only the IPv4 link that needs to be re-established.

Reply:
Really depends on your local network. If you use DHCPv6, this also fixes your IPv6 connectivity ;-).

Reply:
Ping? Still would prefer 

```suggestion
        echo "to keep up-link's Internet connection." >&2
```

here (or alternatively just `IP`).